_posts/2020-09-15-newpipe-0.20.0.md

```diff
@@ -0,0 +1,114 @@
+
+Talking about sharing, when you share a live stream to someone, NewPipe doesn't add a timestamp to the URL any more. This was done by [@nmurali94](https://github.com/nmurali94), thanks.
+
+The header layout has been improved: the alignment on the sides is consistent now and longer headings can fit themselves on two lines instead of just one.
+
+If you ever tried to search something and then lost connection to the internet, or just tried to search while being offline, you would have noticed that NewPipe didn't let you type your search term. Instead it kept on rudely interrupting you with a network error. That is fixed now. You can type all you want, and it will only fail if you actually hit enter or press the search button. Both of these changes were brought to you by [@Stypox](https://github.com/Stypox). Thanks!
+
+[@TobiGr](https://github.com/TobiGr) brought two more improvements into this release: the setting to remember pop-up size and position has a less redundant description, and when you switch to another app, the main player now minimizes to background audio by default to allow for a more seamless transition. Swoosh.
+
+It's 2020, and our default resolutions also realized this. Instead of having 360p everywhere, main player videos are opened in 720p at 60FPS now, and pop-up ones in 480p. Thank you, [@B0pol](https://github.com/B0pol). NewPipe now also remembers what kind of media you downloaded the last time, and will auto-select it when you hit download on a new video. This was done by [@vmazoyer](https://github.com/vmazoyer).
+
+Since 'Autoplay' is a new configurable setting in this release, [@opusforlife2](https://github.com/opusforlife2) changed the 'Autoplay' toggle in the video details to say "Auto-queue" instead. Now it's clearer what that toggle does without leaving room for any confusion.
+
+## Fixes
+
+If you noticed that next to offline search input now working, the actual text of the suggestions look better now, you are correct. [@TobiGr](https://github.com/TobiGr) changed the colours used to the default ones.
+
+An empty error report in certain kinds of crashes is not empty any more, since [@Stypox](https://github.com/Stypox) took care of properly calling code. Yay, code!
+
+Thanks to [@wb9688](https://github.com/wb9688), you can open a video in the local playlist tab, whereas earlier, the app would crash. And thanks to [@BoFFire](https://github.com/BoFFire), if you open the app in the Taqbaylit language, its called `Taqbaylit` now and not `Tamaziɣt Taqbaylit`.
```

Comment:
Those aren't at all related and should have separate paragraphs, same thing below.

Reply:
Previously we put two "unrelated" things in a paragraph as well. I think having two changes per paragraph is a great tradeoff between overall post length and readability. If we would put just one in a paragraph, the length of each section would be too long; if we put more then two in one, users might have the feeling at looking at a wall of text.

That is just a general rule of thumb for me and the posts I write, it changes on a case by case basis ofc, but thats how I used to do it bakc in the day. You younglings.

If more people agree with "one relatable change per paragraph" I will be happy to oblige.

Reply:
I think it's better to have a single change per paragraph as well. But if two changes are related to one another, then it's better to have them in the same paragraph.

Reply:
Im just gonna wait on a third opinion here :(

Reply:
I agree with @opusforlife2, but having a bunch of one sentence paragraphs is also ugly… not sure if it's possible with the current changes to group (semi-)related things together

Reply:
Btw that PR didn't only fix local playlist, but potentially other places as well.

Reply:
added that, the discussion what we put in a paragraph and what not isn't really done yet so I'm gonna leave this as unresolved